File: 4week/rubyClasses_labs/fizzBuzz_lab/fizz_buzz.rb
# Next Module
# Build a class called FizzBuzz that takes two numbers as parameters and then have a method called run that returns a fizzbuzz array (numbers from 1 to 100, numbers divisible by the first number replaced by 'fizz' and numbers divisible by the second number replaced by 'buzz' and numbers divisible by both replaced by 'fizzbuzz'). For instance this code should work with your class:

# fb = FizzBuzz.new(3,5)
# fb.run # returns an array like: [1, 2, 'fizz', 4, 'buzz, ...

class FizzBuzz


    attr_accessor :num1, :num2

    def initialize (num1, num2)
    @num1, @num2 = num1, num2
    end 
    
def run 

    fizzBuzzArray = []
    
    for num in 1..100
        
        if num %num1 == 0 && num %num2 == 0
        fizzBuzzArray.push("FizzBuzz")
            
        elsif num %num1 == 0 
            fizzBuzzArray.push("Fizz")
        elsif num %num2 == 0
            fizzBuzzArray.push("Buzz")
        else  
            fizzBuzzArray.push(num)
        end
    end 
    puts fizzBuzzArray


end 




end 

File: 1week/Day_2/fizzBuzz.js
// take arguments from terminal
const  args = process.argv.slice(2);

//turn args into numbers
let num1 = parseInt(args[0]);
let num2 = parseInt(args[1]);

for (let i = 1; i <= 100; i++) {

//console.log(i);

if (( i % num1 === 0) && (i % num2 === 0)) {
    console.log("fizzbuzz");
}
else if (i % num1 === 0) {
    console.log("fizz");
    //i = "fizz";
} else if (i % num2 === 0) {
    console.log("buzz");
} 

else  { 
    console.log(i);
}

}

// Write a script fizzBuzz.js that takes two numbers, 
// then iterates through the numbers from 1 to 100 logging "fizz" 
// if the current number is divisible by the first number, "buzz" 
// if the current number is divisible by the second number, 
// "fizzbuzz" if it's divisible by both, else print the current number.

